shared-data/pipette/schemas/2/pipetteLiquidPropertiesSchema.json

```
@@ -92,6 +92,11 @@
               "dispense": {
                 "type": "array",
                 "items": { "$ref": "#/definitions/liquidHandlingSpecs" }
+              },
+              "uiMaxFlowRate": {
+                "type": "string",
```

```suggestion
                "type": "number",
```
or probably better is to use a reference to the existing `positiveNumber`